FrontEnd/src/components/ProfilePage/FormComponents/FormFields/HalfFormField.js

```diff
@@ -15,21 +21,23 @@ const HalfFormField = (props) => {
                 </label>
             </div>
             <div className={css['fields__field']}>
-                <input
-                    type={props.inputType ? props.inputType : 'text'}
-                    className={`${css['fields__field--input']} ${props.name === 'email' && css['disabled__field']}`}
-                    name={props.name}
-                    value={props.value}
-                    placeholder={props.fieldPlaceholder ? props.fieldPlaceholder : 'Введіть текст'}
-                    onBlur={props.onBlur}
-                    onKeyDown={preventEnterSubmit}
-                    onChange={props.updateHandler}
-                    required={(props.requredField) ? 'required' : ''}
-                    disabled={(props.name === 'email') ? 'disabled' : ''}
-                    maxLength={props.maxLength}
-                />
+                    <Tooltip title={shouldShowTooltip ? props.value : ''} placement="bottom">
+                        <input
+                            type={props.inputType || 'text'}
+                            className={`${css['fields__field--input']} ${props.name === 'email' && css['disabled__field']}`}
```

Comment:
I would recommend using `classnames` package. You might provide an undesired `false` class name.